### PR TITLE
Portfolio: Mobile Menu

### PR DESCRIPTION
**Pull request by tajulafreen**
In this pull request did the following :

1. Implemented the mobile menu with javascript and made it look like the one in the Figma design template that I choose.
2. Used DOM manipulation to make my mobile version menu look like the template I selected.
3. I used the correct GitHub flow.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+const hambur = document.querySelector('.hamburger');
+const menu = document.querySelector('.menu-section');
+hambur.onclick = function myfunction() {
+  menu.classList.toggle('active');
+  hambur.classList.toggle('active');
+};
```
Comment (AlphaNtihinduka) [inline script.js:14]: - [ ] Please add the functionality such that when the user clicks (or taps) any option of the mobile menu, the mobile menu disappears ☺️.

File: styles.css [hunk 2]
```diff
@@ -831,3 +824,59 @@ ul {
...
+    left: 0;
+    top: 60px;
+    width: 100%;
+    background-color: #fff;
+    transition: 0.5s;
+    height: 450px;
+    overflow: hidden;
+    visibility: visible;
+    opacity: 0;
+  }
```
Comment (AlphaNtihinduka) [inline styles.css]: - [ ] Please kindly consider adding the blur effect. In addition the mobile menu has border radius on bottom. This is to ensure that we stick to the Figma template design and also have a professional portifolio ☺️.

**Review by AlphaNtihinduka — CHANGES_REQUESTED** [review]
## Hello Team🙋‍♂️
Good job working on your mobile menu. You have done the design to the best expectation and incredibly well. 👏  

You still need to implement few on the project but you are almost there!!! 🚶: 

TO HIGHLIGHT:
- The mobile menu disappears when (x) button is clicked ✔️ 
- The mobile menu appears when humberger button is clicked ✔️ 
- Well-structured files ✅

# STATUS: Required Changes :recycle: 

close

# Required Changes :recycle: 
- _Kindly consider the inline review comments._

## Optional suggestions
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or slack channel if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK :crossed_fingers: :four_leaf_clover:

File: script.js (hunk 1 above)
Comment (tajulafreen) [reply script.js:14]: Hi @AlphaNtihinduka 
sorry, I didn't understand could you show me an example for that

File: styles.css (hunk 2 above)
Comment (tajulafreen) [reply styles.css]: Hi @AlphaNtihinduka
what should I add like a blur effect?

File: styles.css [hunk 3]
```diff
@@ -831,3 +824,61 @@ ul {
...
+
+  .menu-section.active {
+    height: 650px;
+    opacity: 1;
+    border-bottom-left-radius: 6px;
+    border-bottom-right-radius: 6px;
+  }
```
Comment (AbelG101) [inline styles.css]: - [ ] [OPTIONAL] THE PREVIOUS REVIEWER FORGOT TO MENTION THIS BUT PLEASE MAKE SURE YOU USE THE PROPER LINTERS SETUP AS THIS MILESTONE REQUIRES YOU TO USE THE "HTML-CSS-JS" CONFIG [FOUND HERE](https://github.com/microverseinc/linters-config/tree/master/html-css-js).
- [ ] As suggested by the previous reviewer please make sure to add a bottom border-radius to the menu as shown in the below Figma design. 👇
- Figma design:
  image
- Your design:
  firefox_Ic39Da92LT
- **HINT:** You should use the `border-bottom-left-radius` and `border-bottom-right-radius`

**Review by AbelG101 — CHANGES_REQUESTED** [review]
# Hi @tajulafreen 👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- UI is looking good ✔
-  Well-structured files ✔
- All linter checks are passing ✔
- Descriptive PR title with a great summary ✔

 ## Required Changes ♻️
- _Check the comments under the review._
- you just have to make a few changes and you're on your way 🚀.

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 3 above)
Comment (tajulafreen) [reply styles.css]: Hi @AbelG101 
I was added but it's not working.

Comment (AbelG101) [reply styles.css]: Hey @tajulafreen, It wasn't working for me. Can you double-check?

**Comment by AbelG101** [comment]
- Hey, @tajulafreen & @Ejikeokegbe I've updated (added) my comment above as the previous reviewer forgot to mention it but it's very crucial so please check it out. It's in all caps.
  image

**Comment by tajulafreen** [comment]
Hi @AbelG101 
I have added the changes that you provided. Kindly check that pls

**Review by Shisui6 — APPROVED** [review]
Hi @tajulafreen, 

# Approved 🪄
Your project is complete! There is nothing else to say other than... it's time to merge it 💎
Congratulations! 🎉 

## Highlights 🎯
- Mobile menu implemented ✅


## Optional suggestions 🪶

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer, and addressing them will really improve your application._

You can also consider the following:

N/A

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**